packages/control-property-editor/src/panels/changes/ControlGroup.tsx

```diff
@@ -31,24 +38,26 @@ export function ControlGroup(controlGroupProps: ControlGroupProps): ReactElement
     const stackName = changes[0].type === SAVED_CHANGE_TYPE ? `saved-changes-stack` : `unsaved-changes-stack`;
     return (
         <Stack>
-            <Stack.Item className={styles.header}>
-                <Link
-                    onClick={(): void => {
-                        const action = selectControl(controlId);
-                        dispatch(action);
-                    }}
-                    style={{
-                        color: 'var(--vscode-textLink-foreground)',
-                        fontSize: '13px',
-                        fontWeight: 'bold',
-                        textOverflow: 'ellipsis',
-                        whiteSpace: 'nowrap',
-                        overflowX: 'hidden',
-                        lineHeight: '18px'
-                    }}>
-                    {text}
-                </Link>
-            </Stack.Item>
+            {text && (
```

Comment:
Since we use a control group for the new 'control' kind, we do not have the `controlName`. And `controlName` is taken from `change.selector.type`, which for 'renameLabel' and 'addXML' unfortunately does not exist. Therefore, the only way to use the ControlGroup.tsx for `kind: 'control'` is to hide the text stack item (group header) because we do not have one for this change kind.

<img width="1114" alt="Screenshot 2024-10-18 at 14 42 34" src="https://github.com/user-attachments/assets/0d145663-37d5-4349-bcbd-a969466ef415">

Maybe, it does not make sense to process this change in the `ControlGroup`; maybe we need a third group like `ControlChange`, then the ChangeStack.tsx will have three options for rendering the change.

<img width="759" alt="Screenshot 2024-10-18 at 14 49 35" src="https://github.com/user-attachments/assets/739fafeb-89ba-46a4-a0db-0a8fc7295455">

Reply:
Fixed with: 643100cb9b23ca46f6bb0ceaec979a495750deb0